ui/analyse/src/view.ts

```diff
@@ -228,8 +229,10 @@ function controls(ctrl: AnalyseCtrl) {
 }
 
 function forceInnerCoords(ctrl: AnalyseCtrl, v: boolean) {
-  if (ctrl.data.pref.coords == 2)
+  if (ctrl.data.pref.coords == 2){
```

Comment:
should be on `1` not `2`

Reply:
It's a function that force inner coordinates, so it's triggered only when coordinates are out of the board. Isn't that the normal behaviour?

Reply:
ah ok